lib/elixir/lib/calendar/naive_datetime.ex

```diff
@@ -720,23 +721,41 @@ defmodule NaiveDateTime do
       iex> NaiveDateTime.to_iso8601(~N[2000-02-28 23:00:13.001])
       "2000-02-28T23:00:13.001"
 
+      iex> NaiveDateTime.to_iso8601(~N[2000-02-28 23:00:13.001], :extended, 0)
+      "2000-02-28T23:00:13.001Z"
```

Comment:
It cannot return "Z" because Z implies Etc/UTC and we have no idea if this naive datetime is actually in UTC. I think it should simply show nothing (i.e. same as above).

Reply:
It needs to be able to return `Z` for reversibility, plus in ISO8601 `Z` is the same as `+00:00` (as it has no timezone information).